contracts/package.json

```diff
@@ -31,7 +31,7 @@
     "build": "tsc -p tsconfig.build.json",
     "postbuild": "cp -r ./artifacts ./build",
     "types": "tsc -p tsconfig.json --noEmit",
-    "docs": "hardhat docgen",
+    "typedoc": "hardhat docgen",
```

Comment:
I guess this works so there's only 1 command to run from the monorepo root, though it's not typedoc but solidity docgen 🤔

Reply:
Right, there is a build command in vercel and I'd keep it simple without `&&`